solutions/kubernetes/helm/chartrepo/nginx-repo/Readme.md

```diff
@@ -237,7 +260,7 @@ helm repo update <repo_name>
 
 #### **Remove and add the repository**
 
-If the update not worked then remove the repository and add it again
+If update not worked then remove the repository and add it again
 
 ```
 helm repo remove <repo_name>
```

Comment:
Please add command to add repo again here.

Reply:
Done